ui/src/pages/accounts/ConfirmEmail/ConfirmEmail.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { Grid, Segment, Label } from 'semantic-ui-react';
+import { Loader } from '@components';
+import { parse } from '../utils';
+
+const Confirmed = ({ isConfirmed }) => {
+  console.log(isConfirmed);
+
+  return isConfirmed ? (
+    <Label color="green">Your email has been confirmed.</Label>
```

Comment:
Maybe we can display the email that got confirmed

Reply:
The view itself requires for the user to be logged in so in general the user knows that the email which was confirmed was his. Do you think you need more information?